pages/projectTeam.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+import Head from "../components/Head";
+import Navbar from "../components/Navbar";
+import Footer from "../components/Footer";
+import { Background, Container, Flex, Column } from "../styles";
+import ButtonWithTextForm from "../components/ButtonWithTextForm";
+import { useToasts } from "react-toast-notifications";
+
+type Props = {
+  projectTeam: ProjectTeam;
+  allPrizes: Prize[];
+};
+
+const ProjectTeam = (props: Props) => {
+  const [team, setTeam] = useState(props.projectTeam);
+  const [allPrizes, setAllPrizes] = useState(props.allPrizes);
+  const { addToast } = useToasts();
+  const handleJoinProjectTeam = code => {
+    const res = fetch("/api/projectTeam/join/" + code, {
```

Comment:
Wanna move these functions to `lib/projectTeam.tsx`? Makes it a bit cleaner to read

Reply:
Will in the followup PR